resources/language/en-US.ini

```diff
@@ -283,7 +294,6 @@ links.all_todos = "All To-Dos"
 links.create_idea_board = "<i class='fa fa-plus'></i> Create New Board"
 links.edit_idea_board = "<i class='fa fa-edit'></i> Edit Board"
 links.add_more = "<span class='iconfa iconfa-plus'></span> Add More"
-links.delete_board = "<i class='fa fa-trash'></i> Delete Board"
```

Comment:
Why did you remove that? Was that a duplicate?

Reply:
Yeah. Duplicate.